sdks/python/apache_beam/io/gcp/bigtableio.py

```diff
@@ -39,13 +39,17 @@
 
 from __future__ import absolute_import
 
+import logging
 import apache_beam as beam
 from apache_beam.metrics import Metrics
 from apache_beam.transforms.display import DisplayDataItem
 
+_LOGGER = logging.getLogger(__name__)
+
 try:
   from google.cloud.bigtable import Client
 except ImportError:
+  _LOGGER.error('Error in: from google.cloud.bigtable import Client', exc_info=True)
```

Comment:
Thanks @carlosmarin !

Can you log it as a warning instead? This import error is somewhat likely for users that do not choose to install apache_beam dependencies related to Google Cloud (`pip install apache-beam[gcp]`). A warning will help surface the issue when debuggig, but it should not freak users out.

Reply:
Makes sense. Done - thanks @pabloem!